eng/scripts/dependencies/generate-deps.ps1

```diff
@@ -7,6 +7,11 @@ param(
 mkdir $ProjectRefPath -force | out-null
 $projRefsPath = Join-Path $ProjectRefPath "azure-sdk-projectreferences.props"
 
+$ignoreList = @(
+  "Azure.Monitor.OpenTelemetry.Exporter",
```

Comment:
Please file issues for these and leave a comment here with a link to the issues.